src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -99,6 +139,7 @@ public void ClearWarningForExpression(int expressionID)
             using (rwl.CreateWriteLock())
             {
                 warnings.RemoveAll(w => w.ExpressionID == expressionID);
+                infos.RemoveAll(w => w.ExpressionID == expressionID);
```

Comment:
@aparajit-pratap I am not 100% sure about the usage of these function or if infos need to be reset. But I guess we want to clear the runtime infos every time graph runs. Please correct me if I was wrong.

Reply:
This could be tricky. In the case of geometry scale info messages, we would want it to be associated with node execution, while in other contexts, it may just be a static info message that has nothing to do with the node execution, in which case this may not be applicable.